runtime/frequency/src/lib.rs

```diff
@@ -151,7 +157,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 1,
+	spec_version: 3,
```

Comment:
✅

Reply:
why 3 instead of 2?

Reply:
Rococo is current 2, so we push all 2 to 3 next upgrade and keep them in sync

Reply:
[Rococo Live Version](https://github.com/LibertyDSNP/frequency/blob/440ad4781f78fe1b3e8f6d64898d86921c734ba7/runtime/frequency-rococo/src/lib.rs#L134)

[Mainnet Live Version](https://github.com/LibertyDSNP/frequency/blob/440ad4781f78fe1b3e8f6d64898d86921c734ba7/runtime/frequency/src/lib.rs#L154)